readme.md

```diff
@@ -11,111 +11,94 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 # Table of Contents
 
+- [Documentation](#documentation)
 - [Installation](#installation)
-- [Usage](#usage)
-  - [Terms](#terms)
-  - [Command Line Usage](#command-line-usage)
-    - [spark-bench generate-data](#spark-bench-generate-data)
-    - [Multiple Arguments for Workload Suite](#multiple-arguments-for-workload-suite)
-    - [Parallel Runs](#parallel-runs)
-    - [Multi-Run](#multi-run)
-- [Configuration File](#configuration-file)
+- [Building It Yourself](#building-it-yourself)
+- [Running the Examples From The Distribution](#running-the-examples-from-the-distribution)
+  - [Creating the Distribution Folder](#creating-the-distribution-folder)
+  - [Setting Environment Variables](#setting-environment-variables)
+  - [Running the Examples](#running-the-examples)
+- [Remember, there's more documentation, but it's not in this readme!](#remember-theres-more-documentation-but-its-not-in-this-readme)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
+ 
+## Documentation
+The [docs](docs) folder in this repo has several documents to help you on your spark-bench journey to greatness.
 
-## Installation
+[understanding-spark-bench](docs/understanding-spark-bench.md) Is the best starting point.
+ 
+## Installation 
 
 1. Grab the latest release from here: <https://github.com/ecurtin/spark-bench/releases/latest>.
 2. Unpack the tarball using `tar -xvzf`.
 3. `cd` into the newly created folder.
 4. Modify `SPARK_HOME` and `SPARK_MASTER_HOST` in `bin/spark-bench-env.sh` to reflect your environment. 
```

Comment:
Changed spark-bench-env.sh to spark-bench-env.sh.template and changed up spark-bench.sh to check for spark-bench-env.sh before continuing.